src/models/Settings.js

```diff
@@ -80,6 +80,7 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, config) {
       'features.deviceFingerprinting': ['boolean', false, false],
       'features.hideSignOutLinkInMFA' : ['boolean', false, false],
       'features.hideBackToSignInForReset' : ['boolean', false, false],
+      'features.showCustomizableBackLinkInMFA' : ['boolean', false, false],
```

Comment:
I guess unlike other cases we can get away with using a flag and use use the object ? or you want to keep the flag for convention sake ?

Reply:
you mean not using `features.*` config? I prefer to keep it since that is a very simple/explicit way to turn on/off functionality in the widget